src/server/getters/get-markets-closing-in-date-range.ts

```diff
@@ -1,16 +1,18 @@
-import { Database } from "sqlite3";
+import * as Knex from "knex";
 import { Address, MarketsContractAddressRow } from "../../types";
 
 // Input: Date Range
 // Output: Markets Closing in Range
-export function getMarketsClosingInDateRange(db: Database, earliestClosingTime: number, latestClosingTime: number, universe: Address, limit: number|null|undefined, callback: (err?: Error|null, result?: any) => void): void {
-  let query = `SELECT market_id FROM markets WHERE end_time >= ? AND end_time <= ? AND universe = ? ORDER BY end_time DESC`;
-  const queryParams = [earliestClosingTime, latestClosingTime, universe];
-  if (limit) {
-    query = `${query} LIMIT ?`;
-    queryParams.push(limit);
-  }
-  db.all(query, queryParams, (err?: Error|null, rows?: MarketsContractAddressRow[]): void => {
+export function getMarketsClosingInDateRange(db: Knex, earliestClosingTime: number, latestClosingTime: number, universe: Address, limit: number|null|undefined, callback: (err?: Error|null, result?: any) => void): void {
+
+  let query: Knex.QueryBuilder = db
```

Comment:
Would prefer for this query to be reverted to SQL, unless there's a reason the query builder is needed here.

Reply:
I'm using the query builder for the LIMIT instead of doing the string manipulation.

Agreed its a bit cumbersome.